### DISCUSSION TITLE
RDP Session Configuration

### DISCUSSION THREAD

**Discussion by Rockyb0a** [jens-maus/thinRoot#21] [Q&A] [answered]
Hi,

I just came across this project.  I hesitated a while before creating this post.  But more I read, more I'm interrested to contribute some steps for people in a same situation.  First, I have no buildroot experience but was contributing to Thinstation2.x long time ago.  I think this is a better approach since the project is based upon a supported platform.

The Thinlinc Configuration https://github.com/jens-maus/thinRoot/discussions/18 post was of great help to understand the inner working.  Mainly the tftp configuration.  The main page still refer to a bzImage in a tar.bz2 which was confusing.  Only the .img in the release folder (if you build you image) or the one that are release here on github.  

Obviously, I needed to create my own with the right timezone and locale.

Can you give a example of your `SESSION_RDP_0` paramaters that could be put inside `thinroot.conf.network` inide `$TFTROOT/thinroot/conf` folder.  Not too sure where is this folder path specified.

Hopefully I can contribute my experience on this build environment, configuration, deployment of this project.

Regards,

Martin

**Comment by jens-maus** [comment, marked as answer]
Sorry for the late answer here, but please find some basic documentation and example `/tftpboot` aka `$TFTPROOT` directory layout here:

https://github.com/jens-maus/thinRoot/tree/master/tftpboot

Further documentation on that matter should hopefully arrive soon. Until then, feel free to ask more questions on how to perform a complete PXE/TFTPROOT setup for a thinRoot installation.
